blockstore/apps/api/v1/tests/test_views.py

```diff
@@ -51,6 +54,27 @@ def setUp(self):
 
         self.bundle_version = self.bundle.versions.first()
 
+        self.taxonomy = TaxonomyFactory(
+            name='This Taxonomy'
+        )
+
+        self.tags = [
+            TagFactory(
+                name='tag1',
+                taxonomy=self.taxonomy,
+            ),
+            TagFactory(
+                name='tag2',
+                taxonomy=self.taxonomy,
+            ),
+        ]
+
+        self.entity = EntityFactory(
+            entity_type='xblock',
+            external_id='some-resource-uri',
+        )
+        self.entity.tags = self.tags
```

Comment:
What is the benefit of factory methods over the normal API?
```
self.tagstore = DjangoTagstore()

self.taxonomy = self.tagstore.create_taxonomy("This Taxonomy")
self.tags = [
    self.taxonomy.add_tag('tag1'),
    self.taxonomy.add_tag('tag2'),
]

self.entity = EntityId(entity_type='xblock', external_id='some-resource-uri')
for t in self.tags:
    self.tagstore.add_tag_to(t, self.entity)
```

Reply:
No benefits that I know. I picked it for consistency with the other tests in blockstore and that's a moot point as it's no longer in blockstore. Thanks for the example code :)